requirements/lint.txt

```diff
@@ -14,3 +14,4 @@ pylint-celery==0.3
 prospector==1.1.6.2
 # prospector 1.1.6.2 is not compatible with 2.0.0
 pyflakes<2.0.0
+Pygments==2.3.1
```

Comment:
I put this here because it was required to make the `lint` tests.
This gets repeated here as it was already moved to the `local_docs_build.txt`